release/apparmor.go

```diff
@@ -133,3 +143,55 @@ func AppArmorFeatures() []string {
 	}
 	return appArmorFeatures
 }
+
+// parser probe related code
+type apparmorParserFeature struct {
+	feature string
+	rule    string
+}
+
+var requestedParserFeatures = []apparmorParserFeature{
+	{"unsafe", "change_profile unsafe /**,"},
+}
+
+// tryParser will run the parser on the rule to determine if the feature is
+// supported.
+func tryParser(rule string) bool {
+	parser := "apparmor_parser"
+
+	_, err := exec.LookPath(parser)
+	if err != nil {
+		return false
+	}
+
+	cmd := exec.Command(parser, "--preprocess")
+	stdin, err := cmd.StdinPipe()
```

Comment:
Go just needs an io.Reader for stdin. So this can be simplified to something like: `cmd.Stdin = bytes.NewBufferString(fmt.Sprintf("profile snap-test {\n %s\n}", rule))` and the go func() can go with that change.

Reply:
Ok. Do you want this now or later?